File: src/main/java/com/sport/service/web/models/place/PlaceFilter.java
package com.sport.service.web.models.place;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class PlaceFilter {
    private String district;
    private String subDistrict;
    private String outdoor;
    private String placeType;
}

File: src/main/kotlin/com/sport/service/web/models/place/PlaceFilter.kt
package com.sport.service.web.models.place

import jakarta.validation.constraints.NotBlank

data class PlaceFilter(
    @field:NotBlank(message = "District must be pointed")
    val district: String,
    @field:NotBlank(message = "Sub district must be pointed")
    val subDistrict: String,
    val outdoor: String?,
    val placeType: String?
)